pyciemss/ouu/ouu.py

```diff
@@ -66,19 +66,19 @@ def __init__(
         self,
         model: Callable,
         interventions: Callable[[torch.Tensor], Dict[float, Dict[str, Intervention]]],
-        qoi: Callable,
+        qoi: List[Callable[[Any], np.ndarray]],
         end_time: float,
         logging_step_size: float,
         *,
         start_time: float = 0.0,
-        risk_measure: Callable = lambda z: alpha_superquantile(z, alpha=0.95),
```

Comment:
Nitpick: Could we rename this argument `risk_measures`?

Reply:
We could but I wonder what would be the downstream effects of changing the name here. I am guessing it should be doable...